### PR TITLE
Relay imps

### PR DESCRIPTION
**Pull request by xrviv**
Added Relay Implementations.

### CODE REVIEW

File: _pages/relayimps/søstr.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: søstr
+permalink: /søstr/
```
Comment (Giszmo) [inline _pages/relayimps/søstr.md]: ASCII, only please. In both file name and permalink.

File: _pages/relayimps/søstr.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+web: 
+github: metasikander/s0str
+tags:
+- relayImplementation
+programming language: Rust
```
Comment (Giszmo) [inline _pages/relayimps/søstr.md]: no spaces in frontmatter keys, please.

```suggestion
progLang: Rust
```

File: _pages/relayimps/søstr.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: søstr
+permalink: /søstr/
+web: 
+github: metasikander/s0str
+tags:
+- relayImplementation
```
Comment (Giszmo) [inline _pages/relayimps/søstr.md]: Please rename the folder, too. It's just relays. The relay instances will not be handled here. They are already in https://github.com/Giszmo/nostr.info/blob/master/_data/relays.yml

```suggestion
- relay
```

File: _pages/relayimps/Minds Nostr Relay.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: Minds Nostr Relay
+permalink: /Minds Nostr Relay/
```
Comment (Giszmo) [inline _pages/relayimps/Minds Nostr Relay.md]: No spaces in file names and permalinks!

```suggestion
permalink: /MindsNostrRelay/
```

File: _pages/relayimps/Minds Nostr Relay.md
Comment (Giszmo) [inline _pages/relayimps/Minds Nostr Relay.md:1]: Rename to `mindsNostrRelay.md`

File: _pages/relayimps/Astro.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: Astro
+permalink: /Astro/
+web: 
+github: Nostrology/astro
+tags:
+- relayImplementation
```
Comment (Giszmo) [inline _pages/relayimps/Astro.md]: rename all to just "relay"

```suggestion
- relay
```

File: _pages/relayimps/Minds Nostr Relay.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+web: 
+github: https://gitlab.com/minds/infrastructure/nostr-relay
```
Comment (Giszmo) [inline _pages/relayimps/Minds Nostr Relay.md]: This always has to be only account/repo!

```suggestion
github: infrastructure/nostr-relay
```

File: _pages/relayimps/Nex .md
Comment (Giszmo) [inline _pages/relayimps/Nex .md:1]: "Nex .md"?? Please fix that space! And please look at your own PRs and think what I might say to things you see.

File: _pages/relayimps/NNostr.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+---
+title: NNostr
+permalink: /NNostr/
+web: 
+github: Kukks/NNostr
+tags:
+- relayImplementation
+programming language: C#
```
Comment (Giszmo) [inline _pages/relayimps/NNostr.md]: GitHub is coloring the C differently than the #.

When it looks like this, it probably won't work. Put between "", please.

File: _pages/relayimps/me.untethr.nostr-relay.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+---
+title: me.untethr.nostr-relay
+permalink:
```
Comment (Giszmo) [inline _pages/relayimps/me.untethr.nostr-relay.md]: These permalinks ... If we don't set them, the path could dictate the link but I'm not sure it's configured. Have you tried running this locally?

Maybe we should remove the permalink line from all files and just default to `/relayimps/me.untethr.nostr-relay` in this case.

**Review by Giszmo — CHANGES_REQUESTED** [review]

File: _pages/relayimps/me.untethr.nostr-relay.md (hunk 8 above)
Comment (xrviv) [reply _pages/relayimps/me.untethr.nostr-relay.md]: me.untethr.nostr-relay 
- with permalink - does not show
- with permalink set to /relayimps/me.untethr.nostr-relay/ - does not show

The naming convention of this  relay implementation is not optimal. 
Another option is to just rename it, as we did with /sostr/

Comment (Giszmo) [reply _pages/relayimps/me.untethr.nostr-relay.md]: Please rename it. File and permalink.

File: _pages/relays/Astro.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+- relay 
+platforms: 
+progLang: 
+- Elixir
+- Javascript
+license: MIT
+nips:
+---
+
+An Nostr relay, built using Elixir.
```
Comment (Giszmo) [inline _pages/relays/Astro.md]: "An Nostr"?

```suggestion
A Nostr relay, built using Elixir.
```

```suggestion
A relay, built using Elixir.
```

File: _pages/relays/NNostr.md [hunk 10]
```diff
@@ -0,0 +1,14 @@
+---
+title: NNostr
+permalink: /NNostr/
+web: 
+github: Kukks/NNostr
+tags:
+- relay
+progLang: 
+- C""#
```
Comment (Giszmo) [inline _pages/relays/NNostr.md]: What??

```suggestion
- "C#" 
```

File: _pages/relays/RelayerBasic.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
+---
+title: Relayer Basic
+permalink: /RelayerBasic/
+web: 
+github: fiatjaf/relayer/tree/master/basic
```
Comment (Giszmo) [inline _pages/relays/RelayerBasic.md]: `github: ` has to be set to the owner/repo and nothing else.

```suggestion
web: https://github.com/fiatjaf/relayer/tree/master/basic
github: fiatjaf/relayer
```

File: _pages/relays/expensiveRelay.md [hunk 12]
```diff
@@ -0,0 +1,17 @@
+---
+title: Expensive Relay
+permalink: /expensiveRelay/
+web: 
+github: fiatjaf/relayer/tree/master/expensive
+tags:
+- relay
+progLang: 
+- Go
+license:
```
Comment (Giszmo) [inline _pages/relays/expensiveRelay.md]: It's the same repo. I assume it's the same license.

```suggestion
web: https://github.com/fiatjaf/relayer/tree/master/expensive
github: fiatjaf/relayer
tags:
- relay
progLang: 
- Go
license: Unlicense
```

File: _pages/relays/nostrPostrRelay.md [hunk 13]
```diff
@@ -0,0 +1,14 @@
+---
+title: NostrPostrRelay
+permalink: /nostrPosterRelay/
+web: 
+github: Giszmo/NostrPostr/tree/master/NostrRelay
+tags:
+- relay
+progLang: 
+- Javascript
+- Kotlin
```
Comment (Giszmo) [inline _pages/relays/nostrPostrRelay.md]: That's mine ...

```suggestion
web: https://github.com/Giszmo/NostrPostr/tree/master/NostrRelay
github: Giszmo/NostrPostr
tags:
- relay
progLang: 
- Kotlin
```

File: _pages/relays/nostrRSRelay.md [hunk 14]
```diff
@@ -0,0 +1,17 @@
+---
+title: nostr-rs-relay
+permalink: /nostrRSRelay/
+web: https://sr.ht/~gheartsfield/nostr-rs-relay/
+github: scsibug/nostr-rs-relay
+tags:
+- relay
+progLang:
+- Python3
```
Comment (Giszmo) [inline _pages/relays/nostrRSRelay.md]: Python??

```suggestion
- Rust 
```

File: _pages/relayimps/me.untethr.nostr-relay.md (hunk 8 above)
Comment (xrviv) [reply _pages/relayimps/me.untethr.nostr-relay.md]: renamed to untethr.md

File: _pages/relayimps/Minds Nostr Relay.md (hunk 6 above)
Comment (xrviv) [reply _pages/relayimps/Minds Nostr Relay.md]: ok, error appears because it is gitlab. changing...

File: _pages/relays/nostrRelay.md [hunk 15]
```diff
@@ -0,0 +1,18 @@
+---
+title: nostr_relay
+permalink: /nostr_relay/
+web: https://code.pobblelabs.org/fossil/nostr_relay/index
+github: 
+tags:
+- relay
+progLang: 
+license: 
+- Copyright
```
Comment (Giszmo) [inline _pages/relays/nostrRelay.md]: Copyright is not a license.
`license:` is not an array.
[it](https://code.pobblelabs.org/fossil/nostr_relay/file?name=LICENSE.txt&ci=tip) looks like some copy-left license but not named, so let's leave it out.

```suggestion
license: 
```

File: _pages/relays/nostrRelay.md [hunk 16]
```diff
@@ -0,0 +1,18 @@
+---
+title: nostr_relay
+permalink: /nostr_relay/
+web: https://code.pobblelabs.org/fossil/nostr_relay/index
+github: 
+tags:
+- relay
+progLang:
```
Comment (Giszmo) [inline _pages/relays/nostrRelay.md]:
```suggestion
progLang:
- Python
```

File: _pages/relays/s0str.md [hunk 17]
```diff
@@ -0,0 +1,12 @@
+---
+title: s0str
+permalink: /s0str/
+web: 
+github: metasikander/s0str
+tags:
+- relay
+programming language: Rust
```
Comment (Giszmo) [inline _pages/relays/s0str.md]: You convinced me that progLang should be an array as we might have cases with more than one language:

```suggestion
progLang:
- Rust
```

**Review by Giszmo — CHANGES_REQUESTED** [review]

File: _pages/relays/Astro.md (hunk 9 above)
Comment (xrviv) [reply _pages/relays/Astro.md]: image

Will proofread quoted stuff from now on.

File: _pages/relays/nostrRSRelay.md (hunk 14 above)
Comment (xrviv) [reply _pages/relays/nostrRSRelay.md]: Hmmm, was confused with the other one also named nostr relay. 

image

File: _pages/relays/nostrRelay.md (hunk 16 above)
Comment (xrviv) [reply _pages/relays/nostrRelay.md]: meant the Python for this one I guess.

File: _pages/relays/s0str.md (hunk 17 above)
Comment (xrviv) [reply _pages/relays/s0str.md]: [👍](https://emojipedia.org/thumbs-up/)

**Review by Giszmo — APPROVED** [review]

**Pull request merged**